media/js/mozorg/home/takeover-2016.js

```diff
@@ -19,6 +19,18 @@
     var $optionLabels = $('label[class!="own-amount"]', $donateOptionsSection);
     var $options = $('input[type="radio"]', $donateOptionsSection);
 
+    var locale = $form.data('locale');
+    var currencyCode = $form.data('currency');
+    var numberFormat = new Intl.NumberFormat(locale, {
```

Comment:
Is this going to throw errors in browsers that don't support these functions? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/NumberFormat